ZipBuilder/Sources/ZipBuilder/CocoaPod.swift

```diff
@@ -59,27 +60,17 @@ public enum CocoaPod: String, CaseIterable {
   }
 
   /// The name of the pod in the CocoaPods repo.
-  public var podName: String {
-    switch self {
-    case .googleSignIn: return rawValue
-    default: return "Firebase/\(rawValue)"
-    }
-  }
-
-  /// The minimum supported iOS version.
-  public func minSupportedIOSVersion() -> OperatingSystemVersion {
-    // All ML pods have a minimum iOS version of 9.0.
-    if rawValue.hasPrefix("ML") {
-      return OperatingSystemVersion(majorVersion: 9, minorVersion: 0, patchVersion: 0)
-    } else {
-      return OperatingSystemVersion(majorVersion: 8, minorVersion: 0, patchVersion: 0)
+  public static func podName(pod: String) -> String {
+    if (!pod.starts(with: "Google") && CocoaPod.allCases.map { $0.rawValue }.contains(pod)) {
+      return "Firebase\(pod)"
     }
+    return pod
   }
 
   /// Describes the dependency on other frameworks for the README file.
-  public func readmeHeader() -> String {
-    var header = "## \(rawValue)"
-    if !(self == .analytics || self == .googleSignIn) {
+  public static func readmeHeader(pod: String) -> String {
```

Comment:
Nit: `podName` may be more appropriate here, was reading down below and assumed the type passed in wasn't a String but `PodInfo` (same with the function below)